atomwrappers.py

```diff
@@ -13,6 +14,8 @@ def __hash__(self):
 class CNode(CAtom):
     atom_type = None
     def __init__(self, name):
+        if self.atom_type == "GeneNode":
+            name = get_current_symbol(name)
```

Comment:
Can you move it to CGeneNode constructor? This way it is easier to find all code related to GeneNode in one place.

Reply:
sure